internal/commands/upload_release.go

```diff
@@ -16,35 +17,27 @@ import (
 
 type UploadRelease struct {
 	FS                    billy.Filesystem
-	KilnfileLoader        KilnfileLoader
 	ReleaseUploaderFinder ReleaseUploaderFinder
 	Logger                *log.Logger
 
 	Options struct {
+		flags.Standard
+
 		UploadTargetID string `           long:"upload-target-id" required:"true" description:"the ID of the release source where the built release will be uploaded"`
 		LocalPath      string `short:"lp" long:"local-path"       required:"true" description:"path to BOSH release tarball"`
-
-		Kilnfile       string   `short:"kf" long:"kilnfile" default:"Kilnfile" description:"path to Kilnfile"`
-		Variables      []string `short:"vr" long:"variable" description:"variable in key=value format"`
-		VariablesFiles []string `short:"vf" long:"variables-file" description:"path to variables file"`
 	}
 }
 
 //go:generate counterfeiter -o ./fakes/release_uploader_finder.go --fake-name ReleaseUploaderFinder . ReleaseUploaderFinder
 type ReleaseUploaderFinder func(cargo.Kilnfile, string) (fetcher.ReleaseUploader, error)
 
 func (command UploadRelease) Execute(args []string) error {
-	_, err := jhanda.Parse(&command.Options, args)
+	err := flags.LoadFlagsWithReasonableDefaults(&command.Options, args, command.FS.Stat)
```

Comment:
**thought**: Maybe rename this function to `LoadFlagsWithDefaults` as Reasonable doesn't add a lot of context.